relayer/ops/monitoring/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Build image
+
+FROM ubuntu:20.04 AS build
+
+# OS dependencies
+RUN apt-get update && apt-get install -y wget gcc build-essential git
+
+# Copy source
+
+RUN mkdir -p /starknet-monitoring/relayer
+COPY ./relayer /starknet-monitoring/relayer
+
+# Install golang
+
+RUN wget -c https://dl.google.com/go/go1.18.1.linux-amd64.tar.gz -O - \
+  | tar -xz -C /usr/local \
+  && mkdir -p /go/src /go/bin
+ENV PATH /usr/local/go/bin:$PATH
+
+# Compile binary
+
+WORKDIR /starknet-monitoring/relayer
+RUN go build -o ./monitoring ./cmd/monitoring/*.go
+
+# Production image
+
+FROM ubuntu:20.04
```

Comment:
I would recommend alpine

Reply:
I agree. I added a TODO to update this at a later point. It shouldn't require any other changes. I'll use this https://github.com/smartcontractkit/atlas/blob/master/evm/Dockerfile as an example. Wdyt?

Reply:
I wonder if you need `alpine` and `scratch` could be enough?

Reply:
I'll do this in a separate PR. Wdyt?